[2024-03-16T05:54:57.326+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: study_case_1.load_to_mongodb_task scheduled__2024-03-16T04:00:00+00:00 [queued]>
[2024-03-16T05:54:57.376+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: study_case_1.load_to_mongodb_task scheduled__2024-03-16T04:00:00+00:00 [queued]>
[2024-03-16T05:54:57.382+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-16T05:54:57.624+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): load_to_mongodb_task> on 2024-03-16 04:00:00+00:00
[2024-03-16T05:54:57.650+0000] {standard_task_runner.py:60} INFO - Started process 540 to run task
[2024-03-16T05:54:57.651+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'study_case_1', 'load_to_mongodb_task', 'scheduled__2024-03-16T04:00:00+00:00', '--job-id', '417', '--raw', '--subdir', 'DAGS_FOLDER/pintro_aplikasi_education/study_case1.py', '--cfg-path', '/tmp/tmpp79nngkh']
[2024-03-16T05:54:57.661+0000] {standard_task_runner.py:88} INFO - Job 417: Subtask load_to_mongodb_task
[2024-03-16T05:54:57.766+0000] {task_command.py:423} INFO - Running <TaskInstance: study_case_1.load_to_mongodb_task scheduled__2024-03-16T04:00:00+00:00 [running]> on host f293363f8e27
[2024-03-16T05:54:57.907+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Agus mahari' AIRFLOW_CTX_DAG_ID='study_case_1' AIRFLOW_CTX_TASK_ID='load_to_mongodb_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-16T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-16T04:00:00+00:00'
[2024-03-16T05:54:57.910+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:315: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-03-16T05:54:58.792+0000] {logging_mixin.py:188} INFO - An error occurred: [Errno 2] No such file or directory: '/opt/***/data/aplikasi_table_2024-03-16-4-00-00.csv'
[2024-03-16T05:54:58.794+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-16T05:54:58.821+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=study_case_1, task_id=load_to_mongodb_task, execution_date=20240316T040000, start_date=20240316T055457, end_date=20240316T055458
[2024-03-16T05:54:58.889+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-16T05:54:59.140+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
